openedx/core/djangoapps/user_api/views.py

```diff
@@ -249,6 +251,23 @@ def get(self, request):
         form_desc = FormDescription("post", reverse("user_api_registration"))
         self._apply_third_party_auth_overrides(request, form_desc)
 
+        if configuration_helpers.get_value(
+            'ALLOW_REGISTRATION_FORM_FIELD_OVERRIDE',
+            settings.FEATURES.get('ALLOW_REGISTRATION_FORM_FIELD_OVERRIDE', False)
+        ):
```

Comment:
Same suggestion [as above](https://github.com/open-craft/edx-platform/pull/120/files#diff-06f447d67e696470d5817797bfe9f7b9R161) to use waffle switches instead of feature flags.

Reply:
note: When the upstream PR is prepared, this addition should be put in a separate PR from the profile notification feature.